safe-global/web-core
# fix: Display short address on smaller screens, add sticky table cells

## What it solves

Resolves #614 

## How this PR fixes it

- Table "Actions" are always visible on smaller screens
- Addresses are short on smaller screens

## Other changes

- Adjusted table button sizes to be small according to the Designs
- Change the Spending limit table variant to be consistent with other tables

## How to test it

1. Open the Safe on a smaller screen (< 600px)
2. Navigate to Assets
3. Observe "Send" Buttons in the Table being visible
4. Navigate to Address Book
5. Observe Buttons within the Table being visible
6. Navigate to Settings/Setup
7. Observe Buttons within the Table being visible
8. Navigate to Settings/Spending limits
9. Observe Buttons within the Table being visible

## Screenshots
<img width="454" alt="Screenshot 2022-09-23 at 15 54 29" src="https://user-images.githubusercontent.com/5880855/191976629-8779f09b-1a53-473c-a319-7d64fc05aec4.png">
<img width="489" alt="Screenshot 2022-09-23 at 15 54 49" src="https://user-images.githubusercontent.com/5880855/191976705-2c255dc3-4837-4c2d-a6dc-c7d038e9ec01.png">
